CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2023 EPAM Systems
+# Author: Volodymyr Babchuk <volodymyr_babchuk@epam.com>
```

Comment:
Well, thanks of course, but I am not author here :)

Reply:
Removed